src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -558,8 +559,7 @@ internal async Task<TResult> CallDurableTaskFunctionAsync<TResult>(
                 }
             }
 
-            // TODO: Support for versioning
-            string version = DefaultVersion;
+            string version = ((IDurableOrchestrationContext)this).Version;

```

Comment:
The `version` here is passed to both orchestrations and activities. I don't think we've really thought through how/whether to version activities. Should we isolate the version assignment to just orchestrations to be safe?

Reply:
Agreed, not passing the version to activities anymore.